### ISSUE TITLE
Collapsible Accordion

### ISSUE THREAD

**Issue by Gustl22** [akveo/react-native-ui-kitten#1044]
## 🚀 Feature Proposal

Customizable Component for collapse contents within an accordion.

## Motivation & Example

I want to provide a list of exercises, the user can select. But I want to keep it all on the same page, to let the user browse faster the items. The items should then expandable and collapsible (different defaults each item, customizable content or icon for header and body). 

See [bootstrap](https://getbootstrap.com/docs/4.3/components/collapse/#accordion-example).

Please correct me, if this feature is already implemented somehow.

**Comment by artyorsh** [comment]
Can the [grouped menu](https://akveo.github.io/react-native-ui-kitten/docs/components/menu/overview#menugroup) work for you?

**Comment by Gustl22** [comment]
Haven't tried yet, if this supports custom items instead of `MenuItem`.
Anyway I noticed that I need a custom Header, too, which is not the case in `MenuGroup`.
So it may be better to make my own component.

Can be closed or keep open, if want to implement such feature.
Thanks for the response :)

**Comment by artyorsh** [comment]
@Gustl22 yup, there are too many options on how it should look. Especially when looking at the one you shared. But let's keep it open to see the interest and prioritize some time.

**Comment by Gustl22** [comment]
I think a good approach would be to make a collapsible `Card` component (with animation), so that the header is always shown and the content is hided completely in collapsed state.

BTW: Used [LayoutAnimation](https://docs.expo.io/versions/latest/react-native/layoutanimation/) before _collapse_ state change and adapted heights of Card and Header.

Here a demo of my implementation:
![Card-Accordion](https://user-images.githubusercontent.com/18537755/79919393-987e5680-842e-11ea-9de0-d39eb2a7a16d.gif)

**Comment by yogevlahyani** [comment]
I can't find group menu for version 4.4 (Don't want to upgrade to 5 since it's still alpha)

**Comment by artyorsh** [comment]
@yogevlahyani v4 docs is [here](https://akveo.github.io/react-native-ui-kitten/docs/4.x). See the version switcher at the top of the page.

**Comment by yogevlahyani** [comment]
@artyorsh Yes I know, MenuGroup isn't in the docs or in v4 in general

**Comment by artyorsh** [comment]
@yogevlahyani Yup, because there was no special component for that. This was achieved in a different and less customizable (that's why it was completely reimplemented) way. For v4, see sub-menu example in the [old docs](https://akveo.github.io/react-native-ui-kitten/docs/4.x/components/menu/overview#menu)

**Comment by yogevlahyani** [comment]
@artyorsh Thanks! :)

**Comment by dlwiest** [comment]
Trying to do something similar here. I have a large form that I want to break up into chunks with an accordion-style layout. I tried to accomplish this using MenuGroups, but I can't find a way to customize MenuItem. Is this possible, or have people found a better solution for accordions?

**Comment by johnkmzhou** [comment]
> Trying to do something similar here. I have a large form that I want to break up into chunks with an accordion-style layout. I tried to accomplish this using MenuGroups, but I can't find a way to customize MenuItem. Is this possible, or have people found a better solution for accordions?

I tried adding a View component to the MenuItem description prop but it causes layout issues (e.g. any Text component inside that View will overflow). Probably because the description prop is expecting Text.

**Comment by GuilhermeAmado** [comment]
Any updates on this? I think a collapse/toggle component would be helpful

**Comment by ffMathy** [comment]
I would also like this. MenuGroup is for selectable items. An accordion is just for hiding some details, but not for selecting. It is super common for accordions to appear in UIs. It's such an essential UI component.

**Comment by angelxmoreno** [comment]
Was it decided to not introduce this new component?
